### DISCUSSION TITLE
Brilliant Z-wave add-on

### DISCUSSION THREAD

**Discussion by sujitvasanth** [hassio-addons/addon-zwavejs2mqtt#359] [Ideas]
Hi Thank you for creating a brilliant add-on! for my ubuntu desktop docker installation with supervisor
I have so much more control than my smartthings hub which is now obsolete.

1. It took a little while to setup and it would be really helpful to have a **setup wizard** to stop the trial and error time in getting things working. I case only think of three common scenarios: raspberry pi, virtual machine, ubuntu docker. And setups: mqtt setup: none, via wave-js intergation, via mqttt discovcery (this seems the best to me). Also most users all use the same sticks so would be easy to pick  the settings up as part of a wizard - I have gen5 Aeon USB stick.

2. Would be really helpful to have a dashboard like on zigbee2mqtt see attached photo - for both switches, plugs and sensors this makes testing and debugging much easier 
![zigbee2mqtt dashboard](https://user-images.githubusercontent.com/18464444/162020505-3a885bdc-fbad-429a-8978-2f97752db16f.png)

Once again thanks for the great software

**Comment by frenck** [comment]
Thank but I think you are mixing up stuff. As written in the add-on introduction, this add-on is based on/uses the zwavejs2mqtt project. If you want stuff like this to change, please report it upstream.

From an add-on perspective, there is nothing I can do.

Thanks! 👍 

../Frenck
